### ISSUE TITLE
Rendered HTML has extra newlines for lists, which causes extra whitespace in Confluence

### ISSUE THREAD

**Issue by MattyRad** [kovetskiy/mark#52]
Oh hai mark!

Thanks for this great app! One small issue I've encountered is that rendering lists appears to result in a few extra newlines being added to the resulting HTML, which Confluence renders with a bunch of extra whitespace. For example:

```markdown
<!-- Space: ~twiseau -->
<!-- Title: you are tearing me apart lisa -->

# Hello world

- This
    - Is
        - A
            - List
```

Running the markdown with either the `--compile-only` or `--dry-run` flags yields this HTML:

```html
<h1 id="hello-world">Hello world</h1>

<ul>
<li>This

<ul>
<li>Is

<ul>
<li>A

<ul>
<li>List</li>
</ul></li>
</ul></li>
</ul></li>
</ul>
```

I'm assuming that the newlines after `This`, `Is`, and `A`, are creating a some distracting extra whitespace, like the following:

![Screenshot from 2021-01-12 21-09-57](https://user-images.githubusercontent.com/1143595/104409293-cce4f600-551a-11eb-9f8e-c92bf99a5a35.png)

Whereas without the whitespace it's much more legible:

![Screenshot from 2021-01-12 21-13-10](https://user-images.githubusercontent.com/1143595/104409445-277e5200-551b-11eb-9ba6-72a2ed01b5f4.png)

This is a trivial example, of course, but lists with 200+ lines can become illegible quickly. Any help is appreciated!

**Comment by MattyRad** [comment]
I'm not a Go engineer but I think this does the trick https://github.com/kovetskiy/mark/pull/53

**Comment by kovetskiy** [comment]
Closed by #53.
